kubernetes/samples/scripts/create-weblogic-domain/model-in-image/cm-docker-file/Dockerfile

```diff
@@ -0,0 +1,16 @@
+# Copyright (c) 2021, Oracle and/or its affiliates.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+FROM busybox
+ARG MOUNT_PATH=/common
+ARG USER=oracle
+ARG USERID=1000
+ARG GROUP=root
+ENV MOUNT_PATH=$MOUNT_PATH
+ENV WDT_MODEL_HOME=$MOUNT_PATH/models
+ENV WDT_HOME=${MOUNT_PATH}/weblogic-deploy
```

Comment:
For clarity, can you make WDT_MODEL_HOME and WDT_HOME injected "ARG" values that are externally set to exactly match the values specified in the domain resource (instead of calculated values based off of MOUNT_PATH)? 

Also, rename WDT_HOME to WDT_INSTALL_HOME...  

Also please rename MOUNT_PATH to COMMON_MOUNT_PATH.  

Finally, add a comment to each of the ARG values that explains them?  For example COMMON_MOUNT_PATH/WDT_MODEL_HOME/WDT_INSTALL_HOME should each have a comment that mentions their domain resource equivalent...